tsconfig.json

```diff
@@ -7,15 +7,16 @@
     "rootDir": ".",
     "skipLibCheck": true,
     "target": "es5",
-    "typeRoots": ["./node_modules/@types"]
+    "typeRoots": ["./node_modules/@types"],
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true
```

Comment:
Could you explain these?

Reply:
These are required for the Angular decorators to work.